addons/im_livechat/models/res_users.py

```diff
@@ -54,6 +67,27 @@ def SELF_WRITEABLE_FIELDS(self):
             "livechat_username",
         ]
 
+    @api.depends("livechat_channel_ids", "is_in_call")
+    def _compute_livechat_is_in_call(self):
+        for user in self:
+            # sudo - res.users: checking if user is in call is allowed if the user is member of a live chat channel.
+            user.livechat_is_in_call = user.sudo().is_in_call if user.livechat_channel_ids else None
```

Comment:
Can be done later of course, but we might need to add an extra OR condition with livechat_ongoing_session_count if user over capacity is no longer in the channel?